refactorig(7.12 getting started tutorial) - updates to new 2021.1 release and improvements (#1377)
bonitasoft/bonita-doc: md/create-application.md

Creating the application is the final step of this Getting Started tutorial.

> To create and deploy our first application in our local environment, the easiest way is to use the **[Application descriptor](applicationCreation.md)**. This editor helps you defining the skeleton of an application, as it contains references to resources installed in Bonita Portal and used by a given application. It will also help us deploying the application directly from Bonita Studio. 

Reviewer: ```suggestion
To create and deploy our first application in our local environment, we will use the **[Application descriptor](applicationCreation.md)**. This editor helps you defining the skeleton of an application, as it contains references to resources installed in Bonita Portal and used by a given application. It will also help us deploying the application directly from Bonita Studio. 
```

   ![Application deployment](images/getting-started-tutorial/create-application/create-application.gif)<!--{.img-responsive .img-thumbnail}-->

::: info
The look & Feel can be fully customized by modifying the [layout](layouts.md) and adding a custom [theme](customize-living-application-theme.md).
:::

The application is ready to be used. Now we need to deploy it in Bonita Portal:
1. Click on the icon **Deploy** ![deploy icon](images/getting-started-tutorial/create-application/deploy-icon.png)
1. Click on **Select all** button to make sure everything is deployed: test organization, business data model, process definition, application page
1. Make sure that **Clean BDM database before deployment** is not checked as we want to keep our data for testing
1. Click on **Deploy** button
1. In the **Deploy status** pop up window, click on **Close** button

   ![Application deployment](images/getting-started-tutorial/create-application/application-deployment.gif)<!--{.img-responsive .img-thumbnail}-->
   
Congratulations! You have successfully created your first process and application with Bonita.

::: info
You can verify that the application has been created in Bonita Portal.

> 1. Click on the Portal icon ![Portal icon](images/getting-started-tutorial/create-application/portal-icon.png) in the Bonita Studio tool bar.

Reviewer: ```suggestion
1. Click on the Portal icon ![Portal icon](images/getting-started-tutorial/create-application/portal-icon.png) in Bonita Studio coolbar.
```
